primeorder/src/field.rs

```diff
@@ -317,6 +317,14 @@ macro_rules! impl_mont_field_element_arithmetic {
             }
         }
 
+        impl $crate::elliptic_curve::ops::Invert for $fe {
+            type Output = $crate::elliptic_curve::subtle::CtOption<Self>;
+
+            fn invert(&self) -> $crate::elliptic_curve::subtle::CtOption<Self> {
+                self.invert()
+            }
+        }
```

Comment:
I think this is going to wind up being a breaking change, since it would conflict with the existing impls of `Invert` in existing crate releases of the downstream elliptic curve implementations.

It's a good idea for the next breaking release, though.

Reply:
So I should remove the macro and implement it manually for whatever type it is missing?

Reply:
Yeah, there shouldn't be any missing impls as `Invert` is required in the bounds for `elliptic_curve::Curve::Scalar`

Reply:
But I need it for FieldElement. There were missing implementations for sure

Reply:
Ok fixed